### PR TITLE
fix(ios): removed fb messenger example code.

### PR DESCRIPTION
**Pull request by vijaysingh-axway**

### CODE REVIEW

**Comment by vijaysingh-axway** [comment]
@lokeshchdhry Please merge this after merging https://github.com/appcelerator-modules/ti.facebook/pull/103.

**Comment by keerthi1032** [comment]
@sgtcoolguy  jenkin is failing. iOS module did not get generated to test. can you please take a look.

File: example/app.js
Comment (sgtcoolguy) [inline example/app.js:22]: Don't we still need this guard? I don't think we support the Places API on Android.

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]
The removal of the messenger example is fine. The issue is that the guard for places not to be shown on Android was removed.

**Comment by sgtcoolguy** [comment]
manually merged after adding android guard back into example

**Pull request closed without merging**
